monero-outreach-docs/translations/es/PSA/files/wallet/wallet_es.sla

```diff
@@ -60,19 +64,98 @@
+        </PAGEOBJECT>
+        <PAGEOBJECT OwnPage="0" PTYPE="4" XPOS="417.0032" YPOS="530" WIDTH="273.6" HEIGHT="252" RADRECT="0" FRTYPE="0" CLIPEDIT="0" PWIDTH="1" PCOLOR="None" PCOLOR2="None" TXTFILLSH="90" COLUMNS="1" COLGAP="0" NAMEDLST="" SHADE="100" SHADE2="100" GRTYP="0" ROT="0" PLINEART="1" PLINEEND="0" PLINEJOIN="0" LINESP="13.2" LOCALSCX="1" LOCALSCY="1" LOCALX="0" LOCALY="0" PICART="1" PLTSHOW="0" BASEOF="0" textPathType="0" textPathFlipped="0" FLIPPEDH="0" FLIPPEDV="0" SCALETYPE="1" RATIO="1" PRINTABLE="1" ANNOTATION="0" ANNAME="" TEXTFLOWMODE="0" TEXTFLOW="0" TEXTFLOW2="0" TEXTFLOW3="0" AUTOTEXT="0" EXTRA="0" TEXTRA="0" BEXTRA="0" REXTRA="0" FLOP="0" PFILE="" PFILE2="" PFILE3="" PRFILE="" EPROF="" IRENDER="1" EMBEDDED="1" LOCK="0" LOCKR="0" REVERS="0" TransValue="0" TransValueS="0" TransBlend="0" TransBlendS="0" isTableItem="0" TopLine="0" LeftLine="0" RightLine="0" BottomLine="0" isGroupControl="0" NUMDASH="0" DASHS="" DASHOFF="0" NUMPO="16" POCOOR="0 0 0 0 273.6 0 273.6 0 273.6 0 273.6 0 273.6 252 273.6 252 273.6 252 273.6 252 0 252 0 252 0 252 0 252 0 0 0 0 " NUMCO="16" COCOOR="0 0 0 0 273.6 0 273.6 0 273.6 0 273.6 0 273.6 252 273.6 252 273.6 252 273.6 252 0 252 0 252 0 252 0 252 0 0 0 0 " NUMGROUP="0" GROUPS="" startArrowIndex="0" endArrowIndex="0" OnMasterPage="" ImageClip="" ImageRes="1" Pagenumber="0" isInline="0" fillRule="1" doOverprint="0" gXpos="0" gYpos="0" gWidth="0" gHeight="0" LAYER="0" BOOKMARK="0" NEXTITEM="-1" BACKITEM="-1">
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="3. Considera mantener un monedero en frío para tus XMR los cuales planees gastar o intercambiar. Piensa: Monedero "/>
+            <para LINESP="13.2"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="en frío = cuenta de ahorro, monedero caliente = cuenta corriente."/>
+            <para LINESP="13.2"/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="4. Visita foros en linea y pregunta sobre la variedad de "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="formas de como las personas mantienen segura y protegidas las llaves privadas o la semilla mnemónica. Luego escoge "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="la opción que mejor te sirva. Comienza aquí: https:// getmonero.org/community/hangouts/. Tener una estrategia "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="es mucho mejor que no tener ninguna. ¿Estas notando que "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="la investigación es importante?"/>
+            <para/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="5. ¡Sigue otro tipo de prácticas que recomendamos aquí "/>
```

Comment:
"**Sigue** otro tipo de prácticas que...". The ¡ symbol shouldn't start at the entire paragraph.

Reply:
This is true.

Reply:
done, and made some changes

Reply:
Sigue otro tipo de prácticas que recomendamos aquí 
como, nunca compartas tus llaves privadas, solo utiliza casas de cambio reconocidas, actualiza tu(s) monedero(s) después de cada bifurcación de la red, no hables de cuantos XMR tienes y, ¡siempre busca aprender más!